src/ring_theory/ideal/operations.lean

```diff
@@ -1316,7 +1316,30 @@ lemma quotient_map_comp_mk {J : ideal R} {I : ideal S} {f : R →+* S} (H : J 
   (quotient_map I f H).comp (quotient.mk J) = (quotient.mk I).comp f :=
 ring_hom.ext (λ x, by simp only [function.comp_app, ring_hom.coe_comp, ideal.quotient_map_mk])
 
-/-- `H` and `h` are kept as separate hypothesis since H is used in constructing the quotient map -/
+/-- If `f : R ≃+* S` is a ring isomorphism and `I : ideal R`, then `map f (map f.symm) = I`. -/
+lemma map_of_equiv (I : ideal R) (f : R ≃+* S) : (I.map (f : R →+* S)).map (f.symm : S →+* R) = I :=
+by simp [← ring_equiv.to_ring_hom_eq_coe, map_map]
+
+/-- If `f : R ≃+* S` is a ring isomorphism and `I : ideal R`, then `comap f.symm (comap f) = I`. -/
+lemma comap_of_equiv (I : ideal R) (f : R ≃+* S) :
+  (I.comap (f.symm : S →+* R)).comap (f : R →+* S) = I :=
+by simp [← ring_equiv.to_ring_hom_eq_coe, comap_comap]
```

Comment:
These two should be simp lemmas, the RHS is considerably simpler. Note that these are essentially the `ideal` versions of the `set` lemmas `equiv.symm_image_image` and `equiv.preimage_symm_preimage`, which may or may not be worth mentioning in their docstrings.

A `TODO` comment saying "add `ideal` analogs of the similar nearby lemmas might make sense too.

Reply:
I added the `simp` tag. I have the impression that the other results are already there (note the `I.map` is bigger than the set-theoretic image, so not all results true for sets are true here).